app/src/main/kotlin/hr/asee/android/template/compose/ui/postlogin/home/contents/card/SeekingCard.kt

```diff
@@ -0,0 +1,127 @@
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import hr.asee.android.template.compose.R
+import hr.asee.android.template.compose.ui.common.service.Seeking
+import hr.asee.android.template.compose.ui.postlogin.home.HomeViewModel
+import hr.asee.android.template.compose.ui.theme.DarkGray
+import hr.asee.android.template.compose.ui.theme.Geomanist
+import java.time.format.DateTimeFormatter
+
+
+@Composable
+fun SeekingCard(seeking: Seeking, viewModel: HomeViewModel) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clickable { viewModel.onSeekingClicked() },
+        backgroundColor = MaterialTheme.colors.onPrimary,
+        shape = RoundedCornerShape(15)
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(start = 15.dp, bottom = 8.dp, top = 8.dp)
+        ) {
+            Image(
+                painter = painterResource(
+                    id = if (isSystemInDarkTheme()) R.mipmap.parking_seeking_dark
+                         else R.mipmap.parking_seeking
+                ),
+                contentDescription = stringResource(id = R.string.home_screen_seeking_card_image_content_description),
+                contentScale = ContentScale.Fit,
+                modifier = Modifier
+                    .height(60.dp)
+                    .width(51.dp)
+            )
+
+            Column(modifier = Modifier.padding(top = 8.dp)) {
+                Text(
+                    text = stringResource(id = R.string.home_screen_seeking_card_title_label),
+                    fontFamily = Geomanist,
+                    fontSize = 16.sp,
+                    fontWeight = FontWeight.Bold
+                )
+
+                Row(horizontalArrangement = Arrangement.SpaceBetween) {
+
+                    Column(modifier = Modifier.padding(vertical = 8.dp)) {
+
+                        Text(
+                            text = stringResource(id = R.string.home_screen_seeking_card_start_date_label),
+                            color = DarkGray,
+                            fontFamily = Geomanist,
+                            fontSize = 14.sp,
+                            fontWeight = FontWeight.Bold,
+                            modifier = Modifier.padding(bottom = 2.dp)
+                        )
```

Comment:
Na dosta mjesta koristiš iste parametre za `Text` objekt, možeš napraviti poseban composable tipa `LabelText` i tamo ove common parametre što uvijek ponavljaš (`color`, `fontFamily`, `fontWeight`) postaviš na defaultnu vrijednost u konstruktoru, a ostale parametre ostaviš kako je (hint: otvori definiciju za kreiranje `Text` composable-a i kopiraj sve parametre od tamo, a ove što najčešće koristiš stavi na najčešće korištene vrijednosti)
Ne moraš mijenjati sve gdje si dosad koristio običan `Text`, to možeš ubuduće koristiti ako želiš